src/main/deploy/addon-self-administration.properties

```diff
@@ -1,10 +1,8 @@
-# OSIAM auth server configuration
-org.osiam.auth-server.home=http://localhost:8080/osiam-auth-server
-
-# OSIAM resource server configuration
-org.osiam.resource-server.home=http://localhost:8080/osiam-resource-server
+# Please sync the client data with the ../sql/example_data.sql
```

Comment:
this is now "../sql/client.sql". I changed this [here](https://github.com/osiam/addon-self-administration/pull/112/files#diff-6fd6cb25e5f1cae18884cf8ef42981f8L40), but you moved the whole block upwards.